### ISSUE TITLE
Feature Request: Offer a PHP/htaccess/etc. script that could be added to the tracking domain that routes things back to the main Matomo URL

### ISSUE THREAD

**Issue by KZeni** [jbrule/piwikplugin-TrackingCodeCustomizer#10]
I'd love to be able to customize the tracking domain/URL (ex. serve the tracking from the current site the tracking is being done on) per what's offered by this plugin, but I'd also like that domain to not necessarily need to have Matomo installed on it.

It would be great if there was some way to have a PHP script, .htaccess/nginx/etc. rule, etc. that could be offered up to make whatever's pointing to that custom tracking URL be routed back to the main Matomo install. I know this would mean updating the site for wherever that custom URL is set to so it can't be an automated thing, but it might be nice to have a script (be it PHP/.htaccess/nginx or all of them [my guess is PHP would be the most compatible option]) offered to accommodate this for those that might want it if/when a custom URL is set.

The end goal is that, you could have the tracking code appear to be coming from the site the tracking is on (per the customized URL set here) while it still uses the singular Matomo install. This could help make sure the tracking code doesn't get blocked by tracking/ad blockers.

**Comment by jbrule** [comment]
That does sound nice but is beyond the scope of this plugin.  What you are describing sounds like what the [tracking-proxy](https://github.com/matomo-org/tracker-proxy) project is intended for. This plugin can help you generate tracking codes that target that proxy. I have personally setup reverse proxy solutions using [Apache's mod_proxy](https://httpd.apache.org/docs/2.4/mod/mod_proxy.html) but that gets a bit involved.
